build.gradle

```diff
@@ -81,6 +82,29 @@ task systemtest(dependsOn: 'zipReport', type: Test) {
   }
 }
 
+task startServerInBackground(dependsOn: 'classes', type: com.github.psxpaul.task.JavaExecFork) {
+  main = mainClassName
+  classpath = sourceSets.main.runtimeClasspath
+  args = ['--repo', 'https://github.com/reposense/RepoSense.git', '--view']
+  waitForPort = 9000
+}
+
+task installCypress(type: com.liferay.gradle.plugins.node.tasks.NpmInstallTask) {
+  workingDir = file('frontend/cypress/')
+}
+
+task cypress(dependsOn: 'installCypress', type: com.liferay.gradle.plugins.node.tasks.ExecuteNpmTask) {
+  workingDir = file('frontend/cypress/')
+  args = ["run-script", "debug"]
+}
+
+task uat(dependsOn: ['installCypress', 'startServerInBackground'], type: com.liferay.gradle.plugins.node.tasks.ExecuteNpmTask) {
```

Comment:
Actually, rather than `uat`, since this is more of a regression testing perhaps rename to just `frontendTest`?